Add docs about custom HTTP headers for "spacewalk-repo-sync" (#217)
uyuni-project/uyuni-docs: modules/reference/pages/command-line-tools.adoc

> === Add custom extra HTTP headers

Reviewer: ```suggestion
=== Add Custom Extra HTTP Headers
```

It is possible to add custom HTTP headers to the requests made by [command]``spacewalk-repo-sync`` at the time of repository and package synchronization.

> The custom HTTP headers are defined at [path]``/etc/rhn/spacewalk-repo-sync/extra_headers.conf`` file. They can be defined per "repository name", "channel label" or globally under the "[main]" section:

Reviewer: ```suggestion
The custom HTTP headers are defined in the [path]``/etc/rhn/spacewalk-repo-sync/extra_headers.conf`` configuration file. 
The headers can be defined by repository name, or channel label.
You can also define global headers by putting them in the ``main`` section:
```

----
[testchannel]
X-MY-HEADER-1=VALUE
X-MY-HEADER-2=VALUE

[mychannel]
X-MY-HEADER-3=VALUE
X-MY-HEADER-4=VALUE

[main]
X-GLOBAL-HEADER=VALUE
----

> This can be particularly useful when dealing with RHUI repositories in the public cloud.

Reviewer: ```suggestion
This can be particularly useful when dealing with Red Hat Update Infrastructure (RHUI) repositories in the public cloud.
```

=== Add Custom Extra HTTP Headers

It is possible to add custom HTTP headers to the requests made by [command]``spacewalk-repo-sync`` at the time of repository and package synchronization.

The custom HTTP headers are defined in the [path]``/etc/rhn/spacewalk-repo-sync/extra_headers.conf`` configuration file. 
The headers can be defined by repository name, or channel label.
You can also define global headers by putting them in the ``main`` section:

----
[testchannel]
X-MY-HEADER-1=VALUE
X-MY-HEADER-2=VALUE

[mychannel]
X-MY-HEADER-3=VALUE
X-MY-HEADER-4=VALUE

[main]

> X-GLOBAL-HEADER=VALUE

Reviewer 2: For consistency with the other examples and to make it clear more than one global header is possible, I'd suggest "x-myglobal..." instead of "x-global..."

```suggestion
X-MYGLOBAL-HEADER=VALUE
```
